src/api.ts

```diff
@@ -292,18 +304,94 @@ function generateOptionsString(input?: GetINCHIOptions | GetINCHIExOptions): str
+ *  - -1: InChIKey is valid and non-standard
+ *  - 0: InChIKey is valid and standard
+ *  - 1: InChIKey has invalid length
+ *  - 2: InChIKey has invalid layout
+ *  - 3: InChIKey has invalid version number (not equal to 1)
+ * @example
+ * ```ts
+ * const status = CheckINCHIKey("VNWKTOKETHGBQD-UHFFFAOYSA-N");
+ * console.log(status);
+ * // 0
+ * ```
+ */
 export function CheckINCHIKey(input: string): CheckINCHIKeyReturnCode {
   return INCHIAPI.CheckINCHIKey(input) as CheckINCHIKeyReturnCode;
 }
 
+/**
+ * Check if the string represents valid InChI/standard InChI
+ * @param {string} input source InChI string
+ * @param {boolean} strict (optional) if false, just briefly check for proper layout (prefix, version, etc.)
+ * @returns One amongst -1 , 0 , 1 , 2 , 3 , 4
+ * - -1: InChI is valid and non-standard
+ * - 0: InChI is valid and standard
+ * - 1: InChI has invalid prefix
+ * - 2: InChI has invalid version number (not equal to 1)
+ * - 3: InChI has invalid layout
+ * - 4: Checking InChI through InChI2InChI either failed or produced a result which does not match the source InChI string
+ * @example
+ * ```ts
+ * const status = CheckINCHI("InChI=1S/C3H5NO/c1-2-5-3-4-1/h3H,1-2H2/p+1", false);
+ * console.log(status);
+ * // 0
+ * ```
+ */
 export function CheckINCHI(input: string, strict?: boolean): CheckINCHIReturnCode {
   return INCHIAPI.CheckINCHI(input, strict === true ? 1 : 0) as CheckINCHIReturnCode;
 }
 
+/**
+ * Returns length of the string
+ * @param {string} input An Inchi string
+ * @returns {number} length of the string
+ * @example
+ * ```ts
+ * const length = GetStringLength("VNWKTOKETHGBQD-UHFFFAOYSA-N");
+ * console.log(length);
+ * // 27
+ * ```
+ */
 export function GetStringLength(input: string): number {
   return INCHIAPI.GetStringLength(input);
 }
 
+/**
+ * This function creates structure from InChI string
+ * @param {string} input An Inchi String
+ * @param {object} options (optional) An object containing chosen options as key and their values as true
+ * @returns output is an object containing status and data
+ *  - status - The returned code
+ *  - data - data contains structure created from input (Inchi) string
+ *   -- atom - Array of atom objects which contains various data like atom co-ordinates (x,y,z) , neighbor, bondType , bondStereo, elName, numBonds, numIsoh, isotopicMass, radical, charge
+ *   -- stereo0D - array of num_stereo0D 0D stereo elements or NULL
+ *   -- numAtoms - number of atoms in the structure
+ *   -- numStereo0D - number of 0D stereo elements
+ *   -- szMessage - A string containing Error/warning ASCIIZ message
+ *   -- szLog - log-file ASCIIZ string, contains a human-readable list of recognized options and possibly an Error/warn message
+ *   -- warningFlags - A 2d Array conataining warnings , [x][y]:
+ *            -- x=0 => Reconnected if present in InChI otherwise Disconnected/Normal
+ *            -- x=1 => Disconnected layer if Reconn. layer is present
```

Comment:
You can completely remove this comment. JSDoc style comment are processed by the TypeDoc library. TypeDoc is already aware of the various type definition so it'll auto-hyperlink the output type.

![Screenshot 2021-08-31 at 6 34 01 PM](https://user-images.githubusercontent.com/25290212/131507409-0da44d30-3e2a-4204-b151-1090d07bfbab.png)

Focus more on **function description** and **example usage**. Rest should be auto-handled by TypeDoc.